WcaOnRails/app/assets/stylesheets/polls.scss

```diff
@@ -28,3 +28,7 @@
   position: relative;
   display: block;
 }
+
+.checkbox p {
```

Comment:
I'm ok with this, but could you add a comment explaining why this `p` tag even exists? (it's generated by the md call, right?)